seq2science/rules/get_fastq.smk

```diff
@@ -33,6 +33,7 @@ rule run2sra:
         # and remove the top level folder since prefetch will assume we are done otherwise
         mkdir -p {params.outdir}; cd {params.outdir}; rm -r {wildcards.run}
 
+        # TODO: for loop can be removed if we dont see the debug message
```

Comment:
I don't understand?

Reply:
I actually don't understand what the original code does actually haha

Im not sure actually why this lock is here. I guess some filelock issues with many parallel starts. But why only this rule specifically? :thinking: